src/compiler/core.ts

```diff
@@ -2,6 +2,8 @@
 /// <reference path="performance.ts" />
 
 namespace ts {
+    // WARNING: The script `configureNightly.ts` uses a regexp to parse out these values.
+    // If changing the text in this section, be sure to test `configureNightly` too.

```

Comment:
@mihailik You could make a pull request updating both `configureNightly.ts` and `core.ts`, but please don't revert it back to where we had multiple `versionMajorMinor` declarations.